cep-releases.md

```diff
@@ -0,0 +1,95 @@
+<table>
+<tr><td> Title </td><td> Conda Release Schedule </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Ken Odegard &lt;kodegard@anaconda.com&gt; </td></tr>
+<tr><td> Created </td><td> May 20, 2022 </td></tr>
+<tr><td> Updated </td><td> July 20, 2022 </td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Describes a release cadence for all future `conda` versions starting with `22.7.0`.
+
+## Specification
+
+We propose regularly scheduled bimonthly (every two months) releases where the tagging/release occurs during the week (per ISO 8601, Monday through Thursday) of second Monday of the month.
+
+In a nod to our many different kinds of users we will propose a deprecation policy (to be defined in a later CEP) that allows for a slower adoption rate (i.e. users could update every 3-4 months instead).
+
+These objectives will be aided in the adoption of [CalVer](https://calver.org/):
+- `YY`: the major version will be the shortened year (22+)
+- `MM`: the minor version will be the shortened month (1-12)
+- `MICRO`: the micro/patch version is reset to zero every month and incremented for every additional release that occurs during that month (0+)
+
+This scheme will start with the July 2022 release of 22.7.0.
+
+This will result in the following regular release schedule:
+
+| Version | Release Week |
+|---|---|
+| 22.7.0 | 2022-07-11 |
+| 22.9.0 | 2022-09-12 |
+| 22.11.0 | 2022-11-14 |
+| 23.1.0 | 2023-01-09 |
+| 23.3.0 | 2023-03-13 |
+| 23.5.0 | 2023-05-08 |
+| 23.7.0 | 2023-07-10 |
+| 23.9.0 | 2023-09-11 |
+| ... | ... |
+
+> **Note**
+> Despite following a bimonthly release schedule we will permit releases to occur at any time between regular releases for hotfixes, security releases, or other high priority changes that require immediate release.
```

Comment:
```suggestion
> Despite following a bimonthly release schedule, we will permit releases to occur at any time between regular releases for hot-fixes, security releases, or other high priority changes that require immediate release.
```

Reply:
I agree that "hotfix" isn't officially a word, but within tech, it's supposed to be unhyphenated (per [Google](https://www.google.com/search?q=define+hotfix) and [Wikipedia](https://en.wikipedia.org/wiki/Hotfix)).